package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coverage-upload-action",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
this should be a minor bump 
your adding a backward compatible feature 

https://semver.org/

Reply:
this will break ai insights service https://github.com/search?q=org%3Aneofinancial+ignoredUsers+repo%3Aneofinancial%2Fai-insights-service&type=code

update this service or let the owners know that their github action it going to fail until they migrate to the new version

Reply:
I will reach out the the COs

Reply:
[no external users using action field](https://github.com/search?q=path:**/*.yml+ignoredUsers+&type=code)
[PR deprecating from AI-insights](https://github.com/neofinancial/ai-insights-service/pull/107)